lib/img/local.js

```diff
@@ -43,6 +43,22 @@ LocalDriver.prototype = {
         }
       });
     });
+  },
+
+  delete: function localDelete(key) {
+    return new P(function deletePromise(resolve, reject) {
+      var dir = PUBLIC_DIR;
+      logger.debug('delete.start', key);
+      fs.unlink(path.join(dir, name), function(err) {
```

Comment:
Looks like some copy-paste error. Why does `upload` use `name` and `delete` use `key` as a parameter name?